File: app/src/main/java/com/niceplaces/niceplaces/models/Event.java
package com.niceplaces.niceplaces.models;

/**
 * Created by Lorenzo on 05/01/2018.
 */

public class Event {

    private String mID, mPlaceID, mDate, mDescription;

    public Event(String id, String placeID, String date, String description){
        mID = id;
        mPlaceID = placeID;
        mDate = date;
        mDescription = description;
    }

    public String getID(){
        return mID;
    }

    public String getPlaceID(){
        return mPlaceID;
    }

    public String getDate(){
        return mDate;
    }

    public String getDescription(){
        return mDescription;
    }

}

File: app/src/main/java/com/niceplaces/niceplaces/models/Event.kt
package com.niceplaces.niceplaces.models

/**
 * Created by Lorenzo on 05/01/2018.
 */
class Event {
    var iD: String? = null
        private set
    var placeID: String? = null
        private set
    var date: String
        private set
    var description: String
        private set

    constructor(id: String?, placeID: String?, date: String, description: String) {
        iD = id
        this.placeID = placeID
        this.date = date
        this.description = description
    }

    constructor(date: String, description: String) {
        this.date = date
        this.description = description
    }

}